solution/ui/regulations/eregs-vite/src/views/PolicyRepositorySearch.vue

```diff
@@ -149,6 +149,8 @@ watch(searchQuery, async (newSearchQuery, oldSearchQuery) => {
     getDocList(getRequestParams($route.query));
 });
 
+getPartsLastUpdated();
```

Comment:
This wasn't being explicitly invoked and the Related Sections links in the results weren't working.  Whoops!